cmd/markdown.go

```diff
@@ -37,6 +37,7 @@ var mdDocumentCmd = &cobra.Command{
 
 func init() {
 	markdownCmd.PersistentFlags().BoolVar(new(bool), "no-required", false, "do not show \"Required\" column or section")
+	markdownCmd.PersistentFlags().BoolVar(new(bool), "no-sensitive", false, "do not show \"Sensitive\" column or section")
```

Comment:
Why did you only scoped this flag to `markdown` commands and not globally? I mean the same noise can be seen on other output formats as well.

Reply:
In other formats, like YAML or JSON, it seems less troublesome to me since it's just another value in the data format. In markdown, it takes away some horizontal space. Does that make sense?

Reply:
That's true and I completely agree about the issue in markdown format, but for the sake of consistency between formats I think it would be good to have it global. But at the moment there's no easy way to have this flag available to `tfconf.Output` for marshalling.

Let me think about it a bit, at the end of the day we can proceed with your proposed change as is and figure out a cleaner way to make this globally available for other formats.